packages/login_client/lib/src/login_client.dart

```diff
@@ -87,8 +83,12 @@ class LoginClient extends http.BaseClient {
   Future<oauth2.Credentials?> get credentials => _credentialsStorage.read();
 
   /// Restores saved credentials from the credentials storage.
+  ///
+  /// You can provide a custom `tokenEndpoint` to override the one stored in
```

Comment:
This dartdoc is no longer valid. We may mention using `oAuthSettings` for the initialization (the `refresh()` method will still use the endpoint from the `Credentials`, but that's inside the `oauth2` package and there's no nice way around that.